src/server/scripts/Kalimdor/ZulFarrak/instance_zulfarrak.cpp

```diff
@@ -139,7 +139,7 @@ class spell_zulfarrak_unlocking : public SpellScriptLoader
                 std::list<WorldObject*> cagesList;
                 acore::AllWorldObjectsInRange objects(GetCaster(), 15.0f);
                 acore::WorldObjectListSearcher<acore::AllWorldObjectsInRange> searcher(GetCaster(), cagesList, objects);
-                GetCaster()->VisitNearbyObject(15.0f, searcher);
+//                GetCaster()->VisitNearbyObject(15.0f, searcher);
```

Comment:
@azerothcore/developers @Winfidonarleyan @Viste @Rochet2 @Shard-MW @vhiperdev @Nefertumm @Kaev  I need help with these 2 lines. If we uncomment them, they will cause GCC non-PCH build to fail.

Any ideas?

Reply:
Is there an error code or something?

Reply:
@Nefertumm 

![image](https://user-images.githubusercontent.com/75517/93310585-46870480-f805-11ea-8cbd-56149079f96c.png)

Reply:
hmm i dont have this on GCC 10.2.0

Reply:
you need add 
```
#include "GridNotifiers.h"
#include "GridNotifiersImpl.h"```
to instance_zulfarrak.cpp 
and remove #include `"GridNotifiers.h"` from zulfarrak.h